modules/n1ql/pages/n1ql-language-reference/index.adoc

```diff
@@ -28,13 +44,38 @@ Statements:: N1QL statements are categorized into the following groups:
 * xref:n1ql-language-reference/stringfun.adoc[string]
 * xref:n1ql-language-reference/subqueries.adoc[subquery]
 
-Comments::
-N1QL supports block comments with the following syntax:
-+
+=== Comments
+
+N1QL supports block comments and line comments.
+
+==== Block Comments
+
 ----
 /*  [ [text] | [newline] ]+  */
 ----
 
+A block comment starts with `/{asterisk}` and ends with `{asterisk}/`.
+The query engine ignores the start and end markers `/{asterisk}&nbsp;{asterisk}/`, and any text between them.
+
+A block comment may start on a new line, or in the middle of a line after other N1QL statements.
+A block comment may contain line breaks.
+
+There may also be further N1QL statements on the same line after the end of a block comment -- the query engine does _not_ ignore these.
+
+==== Line Comments
+
+[.status]#Couchbase Server 6.5#
```

Comment:
I think this would look slightly less jarring if replaced with a sentence opening, along the lines of "New feature, introduced in Server 6.5..."

Reply:
Agreed